ci/taos/plugins-base/pr-audit-build-android.sh

```diff
@@ -140,10 +141,13 @@ function pr-audit-build-android-run-queue(){
         # www-data    ALL=(ALL) NOPASSWD:ALL
         echo -e "[DEBUG] current folder is $(pwd)."
 
+        # NNStreamer root directory for build.
+        export NNSTREAMER_ROOT=$(pwd)
+
         # Apply the patch for Android build
-        pushd ./api/capi
-        ./modify_nnstreamer_h_for_nontizen.sh
-        popd
+        if ! patch -R --dry-run -sfp1 -i $NNSTREAMER_ROOT/packaging/non_tizen_build.patch; then
```

Comment:
Could you append an annotation in front of **if ! patch -R --dry-run ....** statement for code maintenance?

Reply:
Added.

Reply:
Checked.